src/hu_nmt/data_augmentator/augmentators/subject_object_augmentator.py

```diff
@@ -358,33 +439,66 @@ def is_eligible_for_augmentation(hun_graph: DependencyGraphWrapper, eng_graph: D
         Tests if a sentence (graph) pair is eligible for augmentation
         """
 
+        # Should contain one nsubj and one obj in both languages
```

Comment:
and --> or

Reply:
this still needs fixing

Reply:
it needs both one subject and object to check if they have the same ancestor

Reply:
ah sorry, I remember we had a discussion around changing this logic to only check whether it has a single object OR a single subject, but I see this was not implemented. Please disregard this comment.